affiliates/base/templates/base/home.html

```diff
@@ -17,6 +17,30 @@ <h2 class="site-logo">
 {% endblock %}
 
 {% block content %}
+
+  {% if LANG in settings.FRIENDS_MSG_LOCALES %}
```

Comment:
The thinking is that it goes out in english asap, and we turn on each locale as it is translated. 
Thanks for the settings note I will update accordingly.